api/descriptions/tags/tag_cve_post.md

```diff
@@ -18,6 +18,18 @@ Prisma Cloud ingests all the various distro vulnerability feeds, and normalizes
 
 The **package info** tab shows both source and binary package fields in a vulnerability report.
 
+Refer to the following parameter descriptions:
+- **id**: `Required` Specifies the vulnerability ID.
+- **packageName**: `Required` Specifies the source or the binary package name where the vulnerability is found. 
+Specify the source package name for tagging when the vulnerability is found in the source package and is available. 
+Use the wildcard `*` to apply the tag to all the packages where the vulnerability is found.
+- **resourceType**: Specifies the resource type for tagging where the vulnerability is found. 
```

Comment:
@ssugandh Since this is supposed to the defacto reference for input params, please fully specify what resourceTypes are available to specify. Do this for all other params as well, as required